packages/cozy-client/src/CozyClient.js

```diff
@@ -91,35 +92,41 @@ export default class CozyClient {
   }
 
   async login() {
-    this.registerClientOnLinks()
-
-    for (const link of this.links) {
-      if (link.onLogin) {
-        try {
-          await link.onLogin()
-        } catch (e) {
-          console.warn(e)
+    if (!this.isLogged) {
```

Comment:
nit: `if (this.isLogged) { return }` is clearer IMHO and this way there is no indentation to add.

Reply:
I think we should `console.warn`, it is wrong for the application to try to login twice.